include/measurement_kit/dns/dns.hpp

```diff
@@ -131,10 +167,32 @@ class Message {
   public:
     Message(){};
     Message(std::nullptr_t){};
-    double rtt = 0.0;
+    double rtt = 0.0; /* XXX not honoured by the cares engine */
+
+    /*
+     * XXX The following field is a duplicate of rcode.
+     */
```

Comment:
TODO: deal with this field and more generally make sure fields are used and make sense where they are (after working to integrate c-ares I have additional insight as how DNS packets appear).